all-actors/src/main/java/org/sunbird/response/CertificateResponseV2.java

```diff
@@ -0,0 +1,69 @@
+package org.sunbird.response;
+
+import java.util.Map;
+
+public class CertificateResponseV2 {
```

Comment:
Alternative approach is the following:
1. Elevate the common attributes to CertificateResponse
2. Create v1 and v2 derived classes - those add pdfUrl or qrCodeUrl
Following this way, it looks neat and change to setPdfUrl or setQrCodeUrl will be the difference in the if-else clause.